superset-frontend/spec/javascripts/dashboard/components/PropertiesModal_spec.jsx

```diff
@@ -87,14 +87,14 @@ describe('PropertiesModal', () => {
           const modalInstance = wrapper.find('PropertiesModal').instance();
           modalInstance.setState({
             values: {
-              json_metadata: '{"color_scheme":"foo"}',
+              json_metadata: '{"color_scheme": "foo"}',
```

Comment:
Should we add a test case to demonstrate more complex metadata being formatted with multiple lines?

Reply:
I think that would test the behavior of the json formatting function more than it would test our code.